src/components/HotspotListItem.tsx

```diff
@@ -130,6 +151,39 @@ const HotspotListItem = ({
                     </Text>
                   </>
                 )}
+                {distanceAway !== undefined && (
+                  <Box marginRight="s" flexDirection="row" alignItems="center">
+                    <LocationIcon color="#474DFF" width={10} height={10} />
```

Comment:
Can we get this from the theme? `<LocationIcon color={colors.purpleMain} width={10} height={10} />`